android-project/app/src/main/java/org/libsdl/app/HIDDeviceBLESteamController.java

```diff
@@ -169,17 +169,17 @@ public HIDDeviceBLESteamController(HIDDeviceManager manager, BluetoothDevice dev
         // final HIDDeviceBLESteamController finalThis = this;
         // mHandler.postDelayed(new Runnable() {
         //     @Override
-        //     public void run() {
+        //     void run() {
```

Comment:
```suggestion
        //     public void run() {
```

Reply:
I did this on my phone as a reminder to myself for when I get back on my computer, working on this.

Reply:
Oh, got it. :)